staging/src/k8s.io/apiserver/pkg/util/flowcontrol/metrics/metrics.go

```diff
@@ -223,11 +223,12 @@ var (
 	)
 	apiserverRequestConcurrencyLimit = compbasemetrics.NewGaugeVec(
 		&compbasemetrics.GaugeOpts{
-			Namespace:      namespace,
-			Subsystem:      subsystem,
-			Name:           "request_concurrency_limit",
-			Help:           "Shared concurrency limit in the API Priority and Fairness subsystem",
-			StabilityLevel: compbasemetrics.ALPHA,
+			Namespace:         namespace,
+			Subsystem:         subsystem,
+			Name:              "request_concurrency_limit",
+			Help:              "Nominal number of execution seats configured for each priority level",
+			DeprecatedVersion: "1.28.0",
```

Comment:
Can we add a comment to not remove this metric until all supported versions support the new metric?

Reply:
(same for concurrency_in_use)

Reply:
Done.

Reply:
IIRC deprecated metrics are no longer available by default (you need to explicitly enable them).

Reply:
xref: https://kubernetes.io/docs/concepts/cluster-administration/system-metrics/#show-hidden-metrics

So we may actually want to bump the deprecatedVersion to a higher one...

Reply:
I set it to release 1.30, which should be more than enough.  When release 1.30 is the latest, the oldest supported release is 1.27 (if I understand correctly).